packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/inputs.tsx

```diff
@@ -22,8 +22,9 @@ const Inputs = ({
     handleChange,
     onChangeInputValue,
     setCurrentFocus,
-}: TInputs) =>
-    is_input_field ? (
+}: TInputs) => {
+    const is_mobile = isMobile();
+    return is_input_field ? (
```

Comment:
```suggestion
}: TInputs) =>  is_input_field ? (
```

Reply:
@prince-deriv I think would be better to save this line: `const is_mobile = isMobile();`